elastalert/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# sha256 as of 2020-10-06 for buster-slim
+FROM debian@sha256:8d81110c3f93a777e3f4053a6b18b70e4a1003655b8c2664bdf18b19043f99d9
+LABEL maintainer="Freedom of the Press Foundation"
+LABEL description="Elastalert container based on Debian stable"
+ENV TZ "America/Los_Angeles"
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y git python3 python3-pip ca-certificates && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# Fixes SSL verification problem with Slack notifications, see
+# https://github.com/Yelp/elastalert/issues/2955
+# https://github.com/Yelp/elastalert/pull/2760
+# https://github.com/JeffAshton/elastalert/commit/340ce3626b80be791416b7af7176ec110d2a3815
+RUN git clone https://github.com/JeffAshton/elastalert.git /tmp/elastalert-git
+WORKDIR /tmp/elastalert-git
+# RUN git checkout fixing-slack-ssl-verification
+RUN git checkout 340ce3626b80be791416b7af7176ec110d2a3815
+RUN python3 setup.py install
+
```

Comment:
Might be worth doing this in one command that cleans up the clone dir post-install, as we do with `apt-get`; otherwise `/tmp/elastalert-git/.git` lives in a layer in the image (I don't know how big it is).

Reply:
True, I often leave multiple layers in a Dockerfile, mostly because during rapid development I want the caching, but also because they're a bit more readable. Your point is well taken, though: the resulting image is plenty fatter than it needs to be. I'm assuming we're going to circle back here in the near future to _remove_ the forked copy, and default to whatever upstream version has the patch merged, so let's definitely optimize when that happens!